### DISCUSSION TITLE
央行確定升息半碼！總裁楊金龍期末考四大重點一次看

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#916] [General]
15號的時候央行升息了半碼，我覺得是在等選舉結束，才不會損失選票，最近有房貸的尤其是買了很多間房的投資客近期應該生活的有點拮据，因為房貸是浮動利率，會跟著央行升息連動，再加上物價上漲，所以真的是要先想好才去買房借錢。

網址：https://www.cw.com.tw/article/5124076

**Comment by CPCpyt** [comment]
畢竟在台灣是沒有所謂固定利率債券這樣的東西，
任何利率的變動都會對貸款造成巨大的影響，
這也可能帶領台灣房市近年將進入下行的行情，
比起借錢買房，也許在這個時候進場債券會是一個好機會。
